integration_tests/androidx_test/src/sharedTest/java/org/robolectric/integrationtests/axt/EspressoTest.java

```diff
@@ -223,4 +226,29 @@ public void clickButton_after_swipeUp() {
       activityScenario.onActivity(action -> assertThat(action.buttonClicked).isTrue());
     }
   }
+
+  @Test
+  public void focusButton_after_DpadDown() {
```

Comment:
**focusButton_after_DpadDown**: This test verifies the expected focus shift behavior when the DPAD_DOWN key is pressed twice. It operates under the default graphics mode and should pass as expected.